test/sinon-test.js

```diff
@@ -115,7 +115,18 @@ describe("sinon module", function () {
 
         describe("useFakeXMLHttpRequest", function () {
             it("should be the fakeXhr.useFakeXMLHttpRequest export from nise", function () {
-                assert.equals(sinon.useFakeXMLHttpRequest, fakeNise.fakeXhr.useFakeXMLHttpRequest);
+                // use full sinon for this test as it compares sinon instance
+                // proxyquire changes the instance, so `actual instanceof Sandbox` returns `false`
+                // see https://github.com/sinonjs/sinon/pull/1586#issuecomment-354457231
+                sinon = require("../lib/sinon");
+
+                var nise = require("nise");
+
+                sinon.spy(nise.fakeXhr, "useFakeXMLHttpRequest");
```

Comment:
Everything here and above should be in the setup (`beforeEach`).

Reply:
Setup moved to `beforeEach`